Jenkinsfile

```diff
@@ -999,7 +999,7 @@ pass = ${OBS_CREDENTIALS_PASSWORD}
         failure {
             // Send mail, but only if we're develop or master
             script {
-                if ((params.NOTIFICATION_EMAIL != null) && (getBuildType() in [BuildType.Master, BuildType.Develop])) {
+                if ((params.NOTIFICATION_EMAIL != null) && (getBuildType() in [BuildType.Master, BuildType.Develop, BuildType.Nightly])) {
```

Comment:
Are we intentionally not sending mail for RCs?

Or should we just say "send mail unless it's PR or unknown"?

Reply:
Inverted from requesting wanted types to excluding unwanted types.
Commit 8752a8788232f8aabebfb9cac9e721061c2e96ea